### ISSUE TITLE
Argo apidocs no longer served

### ISSUE THREAD

**Issue by laymain** [argoproj/argo#3697]
## Summary 

Argo server UI does not provide API documentation as mentioned in the [documentation](https://argoproj.github.io/argo/rest-api/).
Going to http://localhost:2746/apidocs just renders a blank page (with left menu panel).

## Diagnostics

What version of Argo Workflows are you running?

```yaml
λ kubectl -n argo describe po argo-server-85bb655c4f-dp24h
Name:           argo-server-85bb655c4f-dp24h
Namespace:      argo
Priority:       0
Node:           kind-control-plane/172.18.0.2
Start Time:     Fri, 31 Jul 2020 10:55:25 +0200
Labels:         app=argo-server
                pod-template-hash=85bb655c4f
Annotations:    <none>
Status:         Running
IP:             10.244.0.5
Controlled By:  ReplicaSet/argo-server-85bb655c4f
Containers:
  argo-server:
    Container ID:  containerd://f3fba6d14fe75e17e5ca462c72f47ac9df49a2c8e464671c84f4e69e23afa161
    Image:         argoproj/argocli:v2.9.3
    Image ID:      docker.io/argoproj/argocli@sha256:4353763dc8d748a6e922c81695af6b92524691053df9e6dc19618c2e5bfa7832
```
---

**Message from the maintainers**:

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by alexec** [comment]
What version of Argo Workflows are you running? You must be running > v2.10

**Comment by laymain** [comment]
I'm running the 2.9.3 version, the documentation mentions
> v2.5 and after

**Comment by alexec** [comment]
Available for testing in [v2.10.0-rc1](https://github.com/argoproj/argo/releases/tag/v2.11.0-rc1).
